### PR TITLE
Specify chat2 message format

### PR DESCRIPTION
**Pull request by jm-clius**
Now that [more Waku implementations](https://github.com/status-im/js-waku/issues/15) are interfacing with the `test` cluster and other `chat2` nodes, it has become necessary to specify the `chat2` message format explicitly. This will allow individual implementations to handle messages and their properties (`timestamp`, `nick` and `payload`) consistently.

### CODE REVIEW

File: docs/tutorial/chat2.md [hunk 1]
```diff
@@ -68,4 +68,18 @@ This will bypass the random peer selection process and connect to the specified
...
+
+## `chat2` message protobuf format
+
+Each `chat2` message is encoded as follows
+
+```protobuf
+message Chat2Message {
+  uint64 timestamp = 1;
+  string nick = 2;
+  bytes payload = 3;
```
Comment (D4nte) [inline docs/tutorial/chat2.md:81]: Can we assume the payload is utf-8?

File: docs/tutorial/chat2.md [hunk 2]
```diff
@@ -68,4 +68,18 @@ This will bypass the random peer selection process and connect to the specified
...
\ No newline at end of file
+| `/nick` | change nickname for current chat session |
+
+## `chat2` message protobuf format
+
+Each `chat2` message is encoded as follows
+
+```protobuf
+message Chat2Message {
+  uint64 timestamp = 1;
```
Comment (D4nte) [inline docs/tutorial/chat2.md:79]: This is incorrect, you are using `int64` https://github.com/status-im/nim-waku/blob/b0abbcf5973f085df8802625611471ea56006094/examples/v2/chat2.nim#L63

File: docs/tutorial/chat2.md (hunk 1 above)
Comment (jm-clius) [reply docs/tutorial/chat2.md:81]: Yes, thanks, I will clarify that.

**Review by D4nte — APPROVED** [review]
Confirmed, works as expected!

**Pull request merged**